{{ cookiecutter.project_slug }}/Dockerfile

```diff
@@ -0,0 +1,64 @@
+#
+# Base image with defaults for all stages
+FROM registry.access.redhat.com/ubi9/python-{{ cookiecutter.python_version.replace(".", "") }} AS base
+
+# Keep this version tag in sync with pyproject.toml
+LABEL konflux.additional-tags="{{ cookiecutter.version }}"
```

Comment:
The `version` field is [required](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#version) in the `pyproject.toml`, and I think it would be handy to have an image tagged with this version too. I slightly changed the comment (910f939); everybody could remove it, but I don't die on that hill.

Reply:
fine by me